pkgs/tools/system/pcstat/default.nix

```diff
@@ -1,19 +1,17 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage {
+buildGoModule {
   pname = "pcstat-unstable";
-  version = "2017-05-28";
-
-  goPackagePath = "github.com/tobert/pcstat";
+  version = "2021-09-20";
```

Comment:
Why not use the `0.0.1` version and tag upstream made at that date?

Reply:
Oh, I dont know why I missed that. I have changed it, thank you.